# A light show stack can belong to any of the following groups.
# Only one stack in each group will be active at a time.
# They will be executed in this priority order:
stack_groups:
  - PRE
  - SENSOR      # Computes sensor levels
  - LIGHT       # Computes interactive light levels
  - OPEN_LOOP   # Adds background lights
  - IO          # Records/plays light shows
  - COLOR       # Adjusts the color mapping
  - POST

default_light_show_stacks:
#  - StaticPreLightShowStack
  - SensorProcessorStack
  - LightProcessorStack
#  - OpenLoopLightStack
#  - LightShowBallCreatorStack
#  - LightShowPlane0CreatorStack
#  - ColorProcessorStack
  - ColorBlackWhiteProcessorStack
#  - StaticPostLightShowStack
